src/hooks/data/vaults/useVaultTvl.ts

```diff
@@ -9,12 +10,25 @@ type UseVaultTvlParams = VaultProps & {
 
 export function useVaultTvl({
   asset: _address,
```

Comment:
I went through and set these at one point to be `asset: _address` because the linter does not like when arguments are unused. Just delete the `: _address` part and use it as `asset`:

```tsx
export function useVaultTvl({
  asset,
  vaultAddress,
  poolId,
  type,
}: UseVaultTvlParams) {
  // ...
  const vaultTotalAprFallback = useVaultTotalAprFallback({
    asset,
    vaultAddress,
    enabled: isFallbackEnabled ?? false,
  })
  // ...
}
```

Reply:
done